core/src/main/java/org/mapfish/print/config/Template.java

```diff
@@ -44,6 +44,7 @@
     private String reportTemplate;
     private Map<String, Attribute> attributes = Maps.newHashMap();
     private List<Processor> processors = Lists.newArrayList();
+    private boolean mapExport;
```

Comment:
If `mapExport` is `true`, it would be good to ensure that only one `createMapProcessor` is given. The validation could be added here: https://github.com/mapfish/mapfish-print/blob/master/core/src/main/java/org/mapfish/print/config/Template.java#L273